app/views/layouts/_navigation.html.erb

```diff
@@ -0,0 +1,30 @@
+<nav class="navbar navbar-default">
+  <div class="container-fluid">
+    <!-- Brand and toggle get grouped for better mobile display -->
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+     <%= link_to 'E-learning', root_path, class: 'navbar-brand' %>
+    </div>
+    
+    <% if instructor_signed_in?  %>
+      <%= button_to "Sign out", destroy_instructor_session_path, method: :delete, class: "btn btn-primary"%>
+    <% else %>
+      <%= link_to "Sign up", new_instructor_registration_path, class: "btn btn-primary" %>
+      <%= link_to "Sign in", new_instructor_session_path, class: "btn btn-primary"%>
+    <% end %>
+    
+    <!-- Collect the nav links, forms, and other content for toggling -->
+    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+      <ul class="nav navbar-nav">
+      </ul>
+
+      <ul class="nav navbar-nav navbar-right">
+      </ul>
+    </div><!-- /.navbar-collapse -->
+  </div><!-- /.container-fluid -->
+</nav>
```

Comment:
Add new line here
@Junaid-ahmed58 Please add an empty new line at the end of every file